cmd/main.go

```diff
@@ -287,6 +287,8 @@ func registerApp() *cli.App {
 	registerCmd(configCmd)  // Configure minio client.
 	registerCmd(updateCmd)  // Check for new software updates.
 	registerCmd(versionCmd) // Print version.
+	registerCmd(findCmd)    // Find specific String patterns
```

Comment:
findCmd is already there, we don't need to add it again

Reply:
removed.